src/reducers/rootReducer.ts

```diff
@@ -1,24 +1,44 @@
 import { JWTToken } from '../../types/JWT'
-import { SetTokenAction, SET_TOKEN } from '../actions/types'
+import { RoomId } from '../../types/Room'
+import {
+    ActionTypes,
+    SET_HOST_NAME,
+    SET_ROOM_ID,
+    SET_TOKEN,
+} from '../actions/types'
 
 export interface AppState {
     token: JWTToken | null
+    roomId: RoomId | null
+    hostName: string | null
 }
 
 const initialState = {
     token: null,
+    roomId: null,
+    hostName: null,
 }
 
 const rootReducer = (
     state: AppState = initialState,
-    action: SetTokenAction
+    action: ActionTypes
```

Comment:
as we increase complexity, I think we need to hury up moving out of a rootReducer with all the logic and start splitting the state into multiple one: `authReducer` (or user), `roomReducer`, etc

Reply:
#334